scripts/kind-e2e/e2e.sh

```diff
@@ -190,6 +192,25 @@ function enable_logging() {
     fi
 }
 
+function enable_kubefed() {
+    kubectl config use-context cluster1
+    if kubectl rollout status deploy/kubefed-controller-manager -n ${KUBEFED_NS} > /dev/null 2>&1; then
+        echo Kubefed already installed, skipping setup...
+    else
+        helm repo add kubefed-charts https://raw.githubusercontent.com/kubernetes-sigs/kubefed/master/charts
+        helm install kubefed-charts/kubefed --version=0.1.0-rc2 --name kubefed --namespace ${KUBEFED_NS} --set controllermanager.replicaCount=1 > /dev/null 2>&1
+        for i in 1 2 3; do
+            kubefedctl join cluster${i} --cluster-context cluster${i} --host-cluster-context cluster1 --v=2
+            #master_ip=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' cluster${i}-control-plane | head -n 1)
```

Comment:
Will we need those bits of commented code?, it'd be nice to clean up :)

Reply:
Want to keep this as a placeholder for future @skitt work. He will require it to make e2e to work without dapper.

Reply:
ack then!